devops/create_https_wss_endpoints_aws/master.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain_name": "master.hmny.io",
+  "num_of_shards": 2,
+  "explorers_0" : ["52.54.221.24", "35.174.104.172"],
+  "explorers_1" : ["54.215.237.199", "13.56.250.243"],
+  "explorers_2" : [],
+  "explorers_3" : [],
+  "hosted_zone_id" : "Z2HRLI8TCUZ4AL",
```

Comment:
is it associated with domain_name?  Can you find it from aws?

Reply:
hosted_zone_id is for a step to add entries on route 53. This will be implemented in the next release. no enough time for this feature